eland/operations.py

```diff
@@ -640,8 +689,19 @@ def _groupby_aggs(
 
         body = Query(query_params.query)
 
+        # To return for creating multi-index on columns
+        headers = [field.column for field in agg_fields]
+
+        pd_aggs_copy = pd_aggs.copy()
```

Comment:
Instead of modifying `pd_aggs` and skipping `unpack` w/ count can we instead move the `doc_count` value into the aggregation within `buckets_generator()` (maybe into `_groupby_bucket_count` to ensure we don't conflict with other values?) and then we can use the normal flow of unpacking aggregations?

Let me know if this isn't the case.

Reply:
- My intention was not to pass count to `_unpack`, so that we would use `doc_count` especially for groupby instead of using `count` of `metric_aggs`. 
- The original pd_aggs is used at end for construction of dictionary with all aggs (along with count)

This was needed because, we had common method for both `metric_aggs` and `groupby_aggs` . To eliminate the copy , we can introduce a new parameter `is_Groupby` to skip if 'count' agg is passed to it.

Do you have any better Idea ?

Reply:
Usually doc_count is returned by all bucket by default for all composite aggregation
E.g.: 
```json
"buckets" : [
        {
          "key" : {
            "groupby_Cancelled" : false
          },
          "doc_count" : 11381
        },
        {
          "key" : {
            "groupby_Cancelled" : true
          },
          "doc_count" : 1678
        }
      ]
```
The logic that I have chosen 
is to consider doc_count if `count` is given in pd_agg and not consider finding aggregation on `count` again

- To be honest, I wanted to achieve all in single query

Reply:
I'm going to try pulling the PR locally and playing around.